### ISSUE TITLE
Batch normalization

### ISSUE THREAD

**Issue by rslprpr** [arasmus/ladder#26]
Could some one tell me what will happen if I use normal batch normalization not the one that developed by ladder network? And I have also don't understand why in evaluation process test and validation needs the statistics of train data?

**Comment by hotloo** [comment]
Hi! 

Could you be more specific about what you meant by "normal batch normalization"? 

In evaluation, because the sample normally comes one at a time, the statistics will be noisy. This noise will most likely destroy the network performance. Therefore, one often uses the batch statistics from training data which is much more robust to these type of problem. 

Please let me know if I am not able to answer your question or answered it wrong. Hope this helps!
